src/engines/nwn2/feats.cpp

```diff
@@ -132,6 +136,55 @@ bool Feats::getHasCustomFeat(Custom feat) const {
 	return _hasCustomFeat[feat];
 }
 
+/** Return true only if the creature satisfies the feat requirements */
+bool Feats::meetsRequirements(Creature *creature, uint32 id) const {
+	static const Common::UString kFeatMinCols[] = {"MINSTR", "MINDEX", "MINCON", "MININT", "MINWIS", "MINCHA"};
+	static const Common::UString kFeatMaxCols[] = {"MAXSTR", "MAXDEX", "MAXCON", "MAXINT", "MAXWIS", "MAXCHA"};
+	static const Common::UString kFeatPrereq1 = "PREREQFEAT1", kFeatPrereq2 = "PREREQFEAT2";
+	uint i;
```

Comment:
Move that loop variable into the for-loop, please

Reply:
I've attempted to address your concern. Thanks.